src/main/java/io/elastic/jdbc/QueryBuilders/Query.java

```diff
@@ -396,55 +396,59 @@ protected JsonObjectBuilder addValueToResultJson(JsonObjectBuilder resultBuilder
-                        Direction.OUT, cursorSet.getMetaData().getColumnType(i));
-                  } catch (SQLException e) {
-                    throw new IllegalArgumentException(e);
-                  }
-                })
-                .collect(Collectors.toMap(ProcedureParameter::getName, p -> Utils
-                    .cleanJsonType(
-                        Utils.detectColumnType(p.getType(), ""))));
-
-        while (cursorSet.next()) {
-          JsonObjectBuilder entity = Json.createObjectBuilder();
-
-          params.keySet().forEach(key -> {
-            try {
-              if (cursorSet.getObject(key) == null) {
-                entity.addNull(key);
-                return;
-              }
+        return addResultSetToJson(resultBuilder, cursorSet, name);
+      default:
+        return resultBuilder.add(name, stmt.getString(name));
+    }
+  }
 
-              switch (params.get(key)) {
-                case ("number"):
-                  entity.add(key, cursorSet.getDouble(key));
-                  break;
-                case ("boolean"):
-                  entity.add(key, cursorSet.getBoolean(key));
-                  break;
-                default:
-                  entity.add(key, cursorSet.getString(key));
-                  break;
+  protected JsonObjectBuilder addResultSetToJson(JsonObjectBuilder jsonBuilder, ResultSet resultSet, String name)  throws SQLException{
+    JsonArrayBuilder array = Json.createArrayBuilder();
+
+    Map<String, String> params =
+        IntStream.range(1, resultSet.getMetaData().getColumnCount() + 1)
+            .mapToObj(i -> {
+              try {
+                return new ProcedureParameter(resultSet.getMetaData().getColumnName(i),
+                    Direction.OUT, resultSet.getMetaData().getColumnType(i));
+              } catch (SQLException e) {
+                throw new IllegalArgumentException(e);
               }
-            } catch (SQLException e) {
-              e.printStackTrace();
-            }
-          });
+            })
+            .collect(Collectors.toMap(ProcedureParameter::getName, p -> Utils
+                .cleanJsonType(
+                    Utils.detectColumnType(p.getType(), ""))));
+
+    while (resultSet.next()) {
+      JsonObjectBuilder entity = Json.createObjectBuilder();
+
+      params.keySet().forEach(key -> {
+        try {
+          if (resultSet.getObject(key) == null) {
+            entity.addNull(key);
+            return;
+          }
 
-          array.add(entity.build());
+          switch (params.get(key)) {
+            case ("number"):
+              entity.add(key, resultSet.getDouble(key));
+              break;
+            case ("boolean"):
+              entity.add(key, resultSet.getBoolean(key));
+              break;
+            default:
+              entity.add(key, resultSet.getString(key));
+              break;
+          }
+        } catch (SQLException e) {
+          e.printStackTrace();
```

Comment:
will this error be printed on the frontend? How a user can understand what happened?